infrastructure/lambda/adminApi/app.js

```diff
@@ -495,7 +495,7 @@ export const deleteDeploymentApi = middy().handler(async (event, context) => {
       projectId: projectId,
       microFrontendId: microFrontendId,
     },
-    UpdateExpression: "set activeVersions = :v, #def = :d",
+    UpdateExpression: "set activeVersions = :v, #def = :d remove deploymentId",
```

Comment:
This might look odd with no comma but it's correct!

Reply:
shouldn't ```remove``` being all capital latters?

Reply:
It works the same. See the pre-existing set is lower case also.